core/js/typespecs/prompt.ts

```diff
@@ -44,39 +44,36 @@ export const promptBlockDataSchema = z.union([
 
 export type PromptBlockData = z.infer<typeof promptBlockDataSchema>;
 
-const braintrustModelParamsSchema = z.strictObject({
+const braintrustModelParamsSchema = z.object({
```

Comment:
Maybe we can use `.passthrough()` explicitly instead of reverting to `.object`. The only reason is that we can then enforce that nobody calls `z.object`. Also a comment calling out that this schema is for data received from an external provider would be nice.

Reply:
Sorry I mean `.strip()`. `.passthrough` will actually expose the field in the returned object which I don't think we need.